GNUmakefile

```diff
@@ -84,12 +87,14 @@ cleanest: cleaner
 .PHONY: requirements
 requirements: export CUSTOM_COMPILE_COMMAND='make requirements'
```

Comment:
issue: It no longer makes sense to set this environment variable here. If it is to be kept, it should be moved and updated to the `pin-requirements` target and propagated inside the container run there.

Reply:
Done.